### ISSUE TITLE
Getting Started is difficult

### ISSUE THREAD

**Issue by beautifwhale** [FormidableLabs/spectacle#836]
Just heard about Spectacle, it looks great! I started by reading the docs here https://formidable.com/open-source/spectacle/ but I still cannot get up and running with this at all. I understand there is a rewrite happening which is probably why alot of the links are broken and the information in the docs is no longer relevant. 

Just as an example, If you follow the docs right now there are few things wrong:

- The setup options to get started tell you to run:

`npx create-react-app my-presentation --scripts-version spectacle-scripts`

This is not working at the moment, there is an error due to the `cra-template` not being found. I'm using `node v10`, CRA is not installed globally, using `npx`. So this option is broken.

- The next option redirects you to https://github.com/FormidableLabs/spectacle-boilerplate which is the cli install. This is also not working because of an issue with Fullscreen mode, and it seems like the example is not as feature full as the above demo example here https://raw.githack.com/FormidableLabs/spectacle/master/one-page.html#/. Also I'd love to use Spectacle with CRA because I know CRA. 

- The third option sends you here https://formidable.com/open-source/spectacle/docs/tutorial.md but this link is broken, so thats a no go. 

- The last option requires you to configure the build setup yourself, I don't really want to be messing with/learning the build config to use Spectacle. 

The last thing I tried was switching the repo to v5.8.0 and browsing the readme. The links to Getting Started, and I'm sure a few others, are broken https://github.com/FormidableLabs/spectacle/blob/v5.8.0/docs/getting-started.md. So at the moment I have no stable way to start using/learning Spectacle. 

TLDR; Is there a quick straight forward way for me to create a presentation that uses CRA and provides the examples shown in https://raw.githack.com/FormidableLabs/spectacle/master/one-page.html#/ ?
 

UPDATE: I created a repo with an initial CRA project, installed Spectacle, and added all of the things from the one page demo. https://github.com/beautifwhale/create-react-app-spectacle. Just clone it and go!

**Comment by kale-stew** [comment]
Thanks for opening this issue, @beautifwhale! As you mentioned, some things are moving around in the rewrite which is why the two related repos you mention above are in this mid-flight state. `spectacle-scripts` is being deprecated for v6, and `spectacle-boilerplate` was enhanced to carry the weight of what the scripts offered new users.

Regarding your options to get started:

1. The `npx` script is broken, and for that I apologize. If you were to pin `spectacle-scripts` to a pre-deprecated version ([`v2.1.3`](https://github.com/FormidableLabs/spectacle-scripts/releases/tag/v2.1.3) should work), you would have access to the features from that version.

2. As mentioned above, `spectacle-cli` is undergoing some changes and Fullscreen is broken today (bad luck on both of our parts). `spectacle-cli` differs from `spectacle-boilerplate` in a few ways: (1) It is going to be continually maintained and leveraged as the easiest way to get started with spectacle v6 and later (2) Helps serve or build presentations with existing md/mdx files, instead of providing a "boilerplate" for users to then gut & replace. If you want to check out the boilerplate from before we killed it, you can see the [old presentation/index here](https://github.com/FormidableLabs/spectacle-boilerplate/blob/a0735c34c51895369f6e0c11d57a7c19ae3dcaa0/presentation/index.js).

3. Broken links are the worst. While the hosted Tutorial was taken off the formidable.com site for now, you can see the same guide directly in github, here: https://github.com/FormidableLabs/spectacle/blob/master/docs/content/tutorial.md

4. I also don't recommend configuring your own build step. :-)

TL;DR: I've addressed your points directly above, and hopefully this clarified info will help you (and others in your position today/this week) get started. I can see in your UPDATE you managed to wire a presentation up, so congrats 🎉 and I'm sorry it was made difficult by our [lack of clear] documentation at this time. I will close this issue when the issues above are fixed.

**Comment by migh** [comment]
Hi!

I know it is very hard to handle open-source projects, I've tried and failed myself, but I think better communication could improve things greatly. It is not a complain at all, my intention is to help and it is just a suggestion:

1. I would be glad to contribute, by coding and opening a PR, but I think a better roadmap and prioritization are needed.

2. I open a question and there is no answer or even category added, and that question is repeated on the bottom of the pinned thread. I know you are really busy and I am trying my best to not make it sound as a complain, I can even help with this documentation/issues problem if you point me out in the right direction to do so.

3. I learn about the rewrite effort first when I installed using the mdx mode, I got *v6.0.0-alpha.3* installed.

I loved the idea about the `spectacle boilerplate` try to use it, thought it was easier and had been struggling for the last few days with some things important to me, like:

1. Having a background image for the first slide.
2. A nice way of having customized components for some slides.  (this one I've solved)

I am really frustrated and going back to a previous version with [cra-spectacle-mdx-boilerplate](https://github.com/markerikson/cra-spectacle-mdx-boilerplate) repo for the mdx mode.

TL;DR: I love Spectacle and I am just saying that it is completely normal to be busy and that a rewrite is a big endeavor but better communication, even before (or as a way to) improving documentation would be really appreciated.

**Comment by kale-stew** [comment]
We've finally got our docs up to date, thanks for bearing with us. Please check them out [here](https://github.com/FormidableLabs/spectacle/tree/task/rewrite).
